### PR TITLE
Make buffer size dynamic

### PR DESCRIPTION
**Pull request by qingweibinary**
@borisyankov ok to merge and publish

### CODE REVIEW

**Comment by qingweibinary** [comment]
@borisyankov Opps, wait me update test

File: src/custom.js [hunk 1]
```diff
@@ -94,13 +94,17 @@ export function getDataForContract(
                 const symbol = contract.underlying;
                 if (contract.tick_count) {
                     const start = contract.purchase_time;
-                    const sellT = contract.sell_time;
+                    const sellT = contract.exit_tick_time;
```
Comment (borisyankov) [inline src/custom.js]: Don't be afraid to call it 'sell' or 'sellTime' ;)

**Comment by borisyankov** [comment]
Tests failed

**Comment by qingweibinary** [comment]
Do not merge

**Comment by qingweibinary** [comment]
@borisyankov OK to merge

**Pull request merged**